subsys/net/lib/nrf_cloud/Kconfig

```diff
@@ -216,18 +216,78 @@ config NRF_CLOUD_AGPS_REQ_CELL_BASED_LOC
 	depends on NRF_CLOUD_AGPS_SINGLE_CELL_ONLY
 	default y
 	help
-		When disabled, nRF Cloud will obtain and store the approximate
-		cell-based location but will not send it to the device.
-		When enabled, nRF Cloud will also send the location to the
-		device.
-
-module = NRF_CLOUD_AGPS
-module-str = nRF Cloud A-GPS
-source "subsys/logging/Kconfig.template.log_config"
+	  When disabled, nRF Cloud will obtain and store the approximate
+	  cell-based location but will not send it to the device.
+	  When enabled, nRF Cloud will also send the location to the
+	  device.
 
 endif # NRF_CLOUD_AGPS
 endmenu
 
+menu "nRF Cloud P-GPS"
+
+config NRF_CLOUD_PGPS
+	bool "Enable nRF Cloud Predicted GPS"
+	depends on NRF9160_GPS
+	depends on MODEM_INFO
+	depends on MODEM_INFO_ADD_NETWORK
+	select STREAM_FLASH_ERASE
+	select SETTINGS
+
+if NRF_CLOUD_PGPS
+
+config NRF_CLOUD_PGPS_PREDICTION_PERIOD
+	int "Desired time in minutes between predictions."
+	range 60 480
+	default 240
+	help
+	  This value is used when requesting predicted GPS data from the
+	  cloud. More frequent predictions will result in more accurate
+	  ephemeris values for any given time, but will take more storage
+	  space in flash.
```

Comment:
Is this the ephemeris fit interval? Is some other value currently supported by the cloud than 4 hours? What will happen if some other period is requested?

Reply:
The only practical values using HERE on the cloud side are 120 minutes and 240 minutes.  The cloud (currently) allows multiples of 60 between 120 and 480.  Otherwise it defaults to 240.

Reply:
So the ephemeris fit interval is fixed in 4 hours and if `NRF_CLOUD_PGPS_PREDICTION_PERIOD` is set to 120 minutes, you would get 4 hour ephemerides which are overlapping. Is that right?

I think the description could be made more clear, at least I understand this so that this changes the time each prediction is valid. Would "interval" be a better word than "period" to describe this?